Source/JavaScriptCore/wasm/WasmB3IRGenerator.cpp

```diff
@@ -3912,18 +3912,39 @@ void B3IRGenerator::emitLoopTierUpCheck(uint32_t loopIndex, const Stack& enclosi
     Vector<Value*> stackmap;
     for (auto& local : m_locals)
         stackmap.append(get(local));
-    for (unsigned controlIndex = 0; controlIndex < m_parser->controlStack().size(); ++controlIndex) {
```

Comment:
What is this function used for? Is it dead now that we replaced B3BBQ with the new BBQ?